frida_mode/README.md

```diff
@@ -1,38 +1,40 @@
-# FRIDA MODE
+# FRIDA mode
 
 The purpose of FRIDA mode is to provide an alternative binary only fuzzer for
-AFL just like that provided by QEMU mode. The intention is to provide a very
+AFL++ just like that provided by QEMU mode. The intention is to provide a very
 similar user experience, right down to the options provided through environment
 variables.
 
-Whilst AFLplusplus already has some support for running on FRIDA [here](https://github.com/AFLplusplus/AFLplusplus/tree/stable/utils/afl_frida)
+Whilst AFL++ already has some support for running on FRIDA
+[here](https://github.com/AFLplusplus/AFLplusplus/tree/stable/utils/afl_frida),
 this requires the code to be fuzzed to be provided as a shared library, it
 cannot be used to fuzz executables. Additionally, it requires the user to write
```

Comment:
The link is broken and the paragraph seems to be outdated. Can I delete the paragraph?

Is FRIDA mode still a WIP or is it as far developed as QEMU mode?

Reply:
that was a different frida implementation that I removed. you can just delete the paragraph.

Reply:
There are a few other mentions: https://github.com/AFLplusplus/AFLplusplus/search?q=afl_frida

I can take care of the other MD files.

Reply:
Resolved here. As for the other MD files, I added this to my quality assurance to do and will use a different branch.